images/release/Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV VERSION=1.6 \
 ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 RUN mkdir $TMPDIR && \
-    INSTALL_PKGS="make gcc zip mercurial krb5-devel" && \
```

Comment:
Changes to the release dockerfile need to be replicated to all the specific sub versions.